addons/calendar/static/src/views/attendee_calendar/attendee_calendar_model.js

```diff
@@ -4,11 +4,15 @@ import { _t } from "@web/core/l10n/translation";
 import { CalendarModel } from "@web/views/calendar/calendar_model";
 import { askRecurrenceUpdatePolicy } from "@calendar/views/ask_recurrence_update_policy_hook";
 import { ConfirmationDialog } from "@web/core/confirmation_dialog/confirmation_dialog";
+import { onWillStart } from "@odoo/owl";
 
 export class AttendeeCalendarModel extends CalendarModel {
     setup(params, { dialog }) {
         super.setup(...arguments);
         this.dialog = dialog;
+        onWillStart(async () => {
+            this.defaultDuration = await this.orm.call("calendar.event", "get_default_duration");
+        });
```

Comment:
CalendarModel is not an owl Component so you can't use onWillStart here !

Reply:
Indeed, I moved the rpc to the controller. Is it ok to do 
`this.model.defaultDuration = await this.orm.call("calendar.event", "get_default_duration");` ?

Reply:
It s not a good practice to edit the model value outside of the model
Maybe you should to do it during the model.load ?